docs/Configuration/fleet-server-configuration.md

```diff
@@ -2323,12 +2311,104 @@ See [here](http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html) f
     force_s3_path_style: false
   ```
 
-##### s3_region
+##### s3_software_installers_region
 
 AWS S3 Region. Leave blank to enable region discovery.
 
 Minio users must set this to any nonempty value (eg. `minio`), as Minio does not support region discovery.
 
+- Default value:
+- Environment variable: `FLEET_S3_REGION`
+- Config file format:
+  ```yaml
+  s3:
+    region: us-east-1
+  ```
+
+##### s3_carves_bucket
+
+Name of the S3 bucket for file carves.
+
+- Default value: none
+- Environment variable: `FLEET_S3_BUCKET`
```

Comment:
Same comment applies for the carves config options, env var would be e.g. `FLEET_S3_CARVES_BUCKET` and yaml would be:

```
s3:
  carves_bucket: some-bucket
```